zio-telemetry-example-proxy/src/main/scala/zio/telemetry/example/proxy/Server.scala

```diff
@@ -0,0 +1,72 @@
+package zio.telemetry.example.proxy
+
+import cats.effect.ExitCode
+import io.circe.generic.auto._
```

Comment:
Don't use this one. Rather define the encoders as suggested below, and use `semiauto` or specify instances explicitly.